app/controllers/admin/calendar_events_controller.rb

```diff
@@ -8,6 +8,16 @@ def index
       project = Project.find(params[:project_id])
       authorize project, :show?
       project_filter = {id: project.id}
+    elsif params[:person_id]
+      person = Person.find(params[:person_id])
+      authorize person
```

Comment:
So as we discussed earlier, this is really checking if the current user can `index` `person` which is not what you want.

You really only need a scope here because any user should be able to get their own calendar events, so I would do `skip_authorization`. Or you could add a CalendarEventsPolicy I guess..